bin/BackupPC

```diff
@@ -587,57 +594,54 @@ sub Main_TryToRun_CmdQueue
 {
     my($req, $host);
 
-    while ( $CmdJob eq "" && @CmdQueue > 0 && $RunNightlyWhenIdle != 1
-            || @CmdQueue > 0 && $RunNightlyWhenIdle == 2
-                             && $bpc->isAdminJob($CmdQueue[0]->{host})
-                ) {
+    while ($CmdJob eq "" && @CmdQueue > 0 && $RunNightlyWhenIdle != 1
+        || @CmdQueue > 0 && $RunNightlyWhenIdle == 2 && $bpc->isAdminJob($CmdQueue[0]->{host}) ) {
```

Comment:
@shancock9 Regarding the new feature request https://github.com/perltidy/perltidy/issues/26

We found some bugs here and there with the `--keyword-paren-inner-tightness` flag. Here is an example of what failed. The first whitespace for while is missing.